src/app/modules/assignments/components/AssignmentTimeline/AssignmentTimelineCard.tsx

```diff
@@ -1,33 +1,61 @@
 import * as React from 'react'
-// import {
-//     //Label,
-//     Panel
-// } from 'react-bootstrap'
-// import SheriffAbilityPile from '../../../components/SheriffAbilityPile'
-import { SheriffAssignment } from '../../../../api/index';
+import { 
+    SheriffAssignment,
+    ASSIGNMENT_TYPES 
+} from '../../../../api/index';
```

Comment:
vscode likes to import from `.../index` however you can also import from `../../../../api` i.e. the *index.ts* makes the entire look like a folder.  Not a huge thing, just as an FYI

Reply:
noted and updated :)